sim/.vscode/sbt-launch.sh

```diff
@@ -0,0 +1,12 @@
+#! /usr/bin/env bash
+
+# This script is used by VSCode only to launch sbt and run bloopInstall. It
+# performs no other tasks (compile, run, test, etc..), so I've omitted many
+# of the other command line flags we pass through the make system.
+
+# If we need these to agree we should consider moving stuff into .jvmopts and
+# .sbtopts and out of the Makefiles
+
+set -ex
+source ../env.sh
+java -jar /home/centos/firesim-global-reset/target-design/chipyard/generators/rocket-chip/sbt-launch.jar "$@"
```

Comment:
Bug? 

If we are trying to avoid having to point to the `sbt-launch.jar` we can instead just use the `sbt` command that is provided by conda. Otherwise, I would just make this a relative path.